src/Controller/Api/BlockController.php

```diff
@@ -13,7 +13,8 @@
 
 namespace Sonata\PageBundle\Controller\Api;
 
-use FOS\RestBundle\Controller\Annotations\View;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use FOS\RestBundle\View\View as FOSRestView;
```

Comment:
Why we need this alias?

Reply:
It is much easier to read this be people with less knowlage about `REST` and `nelmio/api-doc`.

Reply:
I think we should not create aliases when they are not required.